src/Backend/Core/Engine/Base/Action.php

```diff
@@ -96,15 +95,15 @@ public function getAction(): string
      */
     public function checkToken(): void
     {
-        $fromSession = (\SpoonSession::exists('csrf_token')) ? \SpoonSession::get('csrf_token') : '';
-        $fromGet = $this->getContainer()->get('request')->query->get('token');
+        $fromSession = BackendModel::getSession()->get('csrf_token', '');
```

Comment:
this also starts a new session if we havent one yet, if we would just do ->getSession that won't work